<ui version="4.0" >
 <class>finderSkinUI</class>
 <widget class="QDialog" name="finderSkinUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find out a new skin</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>59</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>New SVG File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browserButton" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>new</string>
   </property>
  </widget>
  <widget class="QLabel" name="lTitleSkin" >
   <property name="geometry" >
    <rect>
     <x>102</x>
     <y>18</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Choose a skin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
